src/app/_components/NewsletterForm.tsx

```diff
@@ -49,12 +79,19 @@ export function NewsletterForm() {
             error={getError(methods.formState.errors, 'email')}
           />
         </div>
-        <div className="flex min-w-44 [&>*:first-child]:flex-1">
+        <div className="min-w-44 flex-shrink-0 flex-grow">
           <Button type="submit" disabled={isSubmitting}>
             {isSubmitting ? 'Subscribing' : 'Subscribe'}
           </Button>
         </div>
       </div>
+      {isOpen && (
```

Comment:
This line is probably why the toast doesn't transition out of the screen. When `false`, it gets unmounted instead of being animated out.